src/styles/theme.ts

```diff
@@ -18,12 +19,23 @@ const styles = {
 
 const colors = {
   brand: {
-    900: "#1a365d",
-    800: "#153e75",
-    700: "#2a69ac",
+    primary: "#344C5C",
+    secondary: "#40B484",
   },
 };
 
-const theme = extendTheme(styles, config, colors);
+const components = {
+  Button: { ...Navbar },
+};
+
+const overrides = {
```

Comment:
I am ok with this one, but what the difference between
`const theme = extendTheme({ colors: {} })` and 
`const overrides = {colors: {}}. const theme = extendTheme(overrides)`
😂